fern/pages/docs/api-references/v2-parser.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: OpenAPI v2 parser
+subtitle: The next generation of the OpenAPI Parser for Docs
+---
+
+The next generation of the OpenAPI parser was built explicitly for Fern Docs. It's designed to be more expressive, more flexible and enable faster feature
+development.
+
+Previously, the parser was passed through several lossy layers of transformation, resulting in less accurate representations of the OpenAPI specification.
```

Comment:
I think we avoid talking about the past and just focus on the new feature.

Would want to mark this as a "beta" on this page.